metaflow/mflog/__init__.py

```diff
@@ -37,6 +37,13 @@
             ' >> $MFLOG_STDOUT; echo $1; '\
      ' }' % TASK_LOG_SOURCE
 
+BASH_MFLOG_KFP =\
+    'mflog(){{ '\
+        'T=$(date -u -Ins|tr , .); '\
+        'echo \"[MFLOG|0|${{T:0:26}}Z|%s|$T]$1\"'\
+            ' >> $MFLOG_STDOUT; echo $1; '\
+     ' }}' % TASK_LOG_SOURCE
```

Comment:
could you please add comment why this must be different from BASH_MFLOG and what's different?